packages/nextjs/utils/toast.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { CheckmarkIcon, ErrorIcon, toast as hotToast } from "react-hot-toast";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import { ExclamationTriangleIcon, InformationCircleIcon } from "@heroicons/react/24/solid";
+import Spinner from "../components/Spinner";
+
+type TToastProps = {
+  message: React.ReactNode;
```

Comment:
should I rename it from `message` -> `messageOrJSX` ?

Reply:
I think "content" it's a better name. I'll rename it